marketplace/collect.schema.json

```diff
@@ -49,6 +49,43 @@
                             "type": "string"
                         }
                     }
+                },
+                "secondaryEntries": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "maxProperties": 3,
+                    "patternProperties": {
+                        "^[A-Za-z0-9_]": {
```

Comment:
`"^[A-Za-z0-9_]+$"`. You need `+$` in the end, otherwise you also match `""` and `a#$%###!!`